grpc-example/grpc-example-java/project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 // The Lagom plugin
-addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M5")
+addSbtPlugin("com.lightbend.lagom" % "lagom-sbt-plugin" % "1.6.0-M6")
```

Comment:
Not sure about this one. @dwijnand did you leave this particular sample out for incompatibilities with `play-grpc`?

Reply:
Yes.  I suspect it'll need play-grpc to be updated, at some point.

Reply:
Meh, I see now that the error is on the grpc generated code using the deprecated `ActorMaterializer` and this build being tuned to fail on `warn`. It'd be good to have this fixed in `play-grpc`.

Reply:
I've removed this change from the PR and only left the fix for #94

Reply:
Correction: the generated code introducing `ActorMaterializer` is on akka/akka-grpc.